apps/server/src/shared/domain/rules/base.rule.spec.ts

```diff
@@ -19,6 +21,9 @@ class TestRule extends BaseRule {
 describe('base.rule', () => {
 	let orm: MikroORM;
 	let service: BaseRule;
+	const permissionA = 'a' as Permission;
+	const permissionB = 'b' as Permission;
+	const permissionC = 'c' as Permission;
```

Comment:
why don't you take from the ENUM

Reply:
In the test it create new generic permissions 'a','b','c'. I only replace it in similar way. If we remove permissions, we must maybe replace the permissions in this test if we use it from enum. But i can also replace it, if you like.

Reply:
Is ok